hack/go_container.sh

```diff
@@ -30,7 +30,7 @@ SOURCE_DIR="${SOURCE_DIR:-$(pwd -P)}"
 # default to disabling CGO for easier reproducible builds and cross compilation
 export CGO_ENABLED="${CGO_ENABLED:-0}"
 # the container image, by default a recent official golang image
-GOIMAGE="${GOIMAGE:-golang:1.13.1}"
+GOIMAGE="${GOIMAGE:-golang:1.13.3}"
```

Comment:
Please also update references in the docs when updating the version here

Reply:
I checked the docs before submitting this PR. 
Currently we have used `1.13` directly in our docs without a specific minor version, like this:

>README.md
30:**NOTE**: please use the latest go to do this, ideally go 1.13 or greater.

> site/content/docs/user/quick-start.md
20:**NOTE**: please use the latest Go to do this, ideally go 1.13 or greater.

>site/content/docs/contributing/getting-started.md
54:Preferably Go `1.13` or greater should be installed. 

Should I update all of them to specific minor version?

Reply:
hmm you're right, let's leave it for now.